api/Document.json

```diff
@@ -6978,10 +6984,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": false
```

Comment:
This is `onafterscriptexecute`. To confirm, I also grepped a WebKit checkout and its history for "afterscriptexecute" to ensure the event itself hasn't been supported. No matches 👍